packages/speech/src/index.js

```diff
@@ -87,6 +88,7 @@ function Speech(options) {
   };
 
   common.GrpcService.call(this, config, options);
+  this.api = new v1beta1(options).speechApi();
```

Comment:
Yeah, though looking back through https://github.com/GoogleCloudPlatform/google-cloud-node/pull/1476 it makes more sense why it works the way it works. Also, looks like the `new` is unnecessary.